core/test/audits/seo/link-text-test.js

```diff
@@ -10,15 +10,15 @@ import LinkTextAudit from '../../../audits/seo/link-text.js';
 
 describe('SEO: link text audit', () => {
   it('fails when link with non descriptive text is found', () => {
-    const invalidLink = {href: 'https://example.com/otherpage.html', text: 'click here', rel: ''};
```

Comment:
I think adding a new test or two that has non-english text that is bad would be clearer. All these existing test cases could leave off `lang: 'en'` since it doesn't modify the thing they are asserting; plus that gives us coverage for the common case of there being no document language.

Reply:
I cleaned it up and added more tests.